angular-sample-spa/README.md

```diff
@@ -10,22 +10,95 @@ The IBM Cloud AppID SDK can be used with React to create a secure single-page ap
 
 ## To run locally
 
-* Clone the repository
+1 Clone the repository.
 ```
-git clone 
+git clone https://github.com/ibm-cloud-security/appid-sample-code-snippets.git
 ```
-* Navigate to application workspace folder.
+2 Navigate to application workspace folder.
 ```
-cd angular-sample-spa
+cd angular-sample-app
```

Comment:
it doesn't seem like the directory is renamed. I still see it as `angular-sample-spa`

Reply:
i also see `angular-sample-spa`